### PR TITLE
docs: Add 2023-03-23 meeting notes

### PR DESCRIPTION
**Pull request by sam3k**
Closes: https://github.com/eslint/tsc-meetings/issues/417

### CODE REVIEW

File: notes/2023/2023-03-23.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+### [Rule Change: [no-loop-func] don’t warn when control flow will always leave the loop #16954](https://github.com/eslint/eslint/issues/16954)
+
+**Summary:** Propose changing `no-loop-func` to exempt a particular pattern.
+
+> @nzakas:
+>
+> We've always said that we want rules to cover the 90% case, and I think this definitely falls within the 10% case.
+>
+> Okay, we've decided to not accept this proposal as the complexity cost isn't worth it.
```
Comment (nzakas) [inline notes/2023/2023-03-23.md:38]: I think adding a resolution section to match the rest will make this easier to skim:

```suggestion
**Resolution:** We are not going to accept this proposal as the complexity cost is too high.
```

File: notes/2023/2023-03-23.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+**Q&A**
+
+**It seems like the open question is about whether or not to add an option?**
+A: Given the purpose of this rule, I don't think an option is necessary
+
+**Do we want to keep reporting on IIFEs inside of async and generator functions?**
+Yes, we should keep reporting on that, as it's a code that isn't immediately executed
+
+**Resolution:** we have agreed to:
```
Comment (nzakas) [inline notes/2023/2023-03-23.md:66]: Nice. :+1:

**Review by nzakas — COMMENTED** [review]
Thanks, just left one suggestion.

**Comment by sam3k** [comment]
@nzakas I've accepted the change. Thank you. It's ready for you.

**Review by nzakas — APPROVED** [review]
LGTM. @mdjermanovic can you approve?

**Review by mdjermanovic — APPROVED** [review]
Looks good, thanks!

**Comment by nzakas** [comment]
@sam3k it looks like this issue hasn't yet been updated: https://github.com/eslint/eslint/issues/16954

Thanks so much for your help with the meeting notes. It is very much appreciated. 🙏

**Pull request merged**